tests/x11/steam.pm

```diff
@@ -23,8 +24,11 @@ sub run {
     # start steam in terminal to watch the progress on initial startup, e.g.
     # the update progress
     script_run 'steam', 0;
-    assert_and_click 'steam-accept-agreement';
-    assert_and_click 'steam-agreement-ok';
+    if (!is_tumbleweed) {
```

Comment:
Do we have newer version of steam on TW? If yes, I would expect same behavior will reach other distributions, so we could use multi tag assert and if license is shown, we would click ok. This will help us not to touch code when e.g. leap gets same update. Also, I guess we need to be more sure about change, if that's expected not to show license. Otherwise I would question usefulness of the test which we adjust to the product changes and not changes requirements.

Reply:
Yes - Leap has `1.0.0.54` and TW has `1.0.0.56`.

Reply:
Updated the patch so that it will actually check for the needle and decide based on that.

Reply:
> Do we have newer version of steam on TW? If yes, I would expect same behavior will reach other distributions, so we could use multi tag assert and if license is shown, we would click ok. This will help us not to touch code when e.g. leap gets same update. Also, I guess we need to be more sure about change, if that's expected not to show license. Otherwise I would question usefulness of the test which we adjust to the product changes and not changes requirements.

Actually it is the job of Valve to make sure that their license agreement is accepted. As a steam user I don't see any benefit from the license agreement. It is just some bureaucratic crap and we are in a better world without it. I won't report an upstream bug for an application like steam, if they fail to show the license agreement as long as the rest of the application still works.